bower.json

```diff
@@ -1,6 +1,32 @@
 {
-	"name": "Ripple.js",
-	"author": "Jacob Kelley",
-	"version": "1.2.0",
-	"main": "dist/ripple.min.js"
-}
\ No newline at end of file
+  "name": "Ripple.js",
+  "version": "1.2.0",
+  "homepage": "http://jakiestfu.github.io/Ripple.js/demo",
+  "authors": [
+    "Jacob Kelley <jakie8@gmail.com>"
+  ],
+  "description": "Adds Material style ripple to anything",
+  "main": [
+    "src/ripple.js",
+    "src/ripple.css"
+  ],
```

Comment:
Even though the source files happen to be a 1-1 to uncompressed dist files, I'd still prefer to reference the files in the dist folder instead. That being said, if we ever decide to use less as a src, this approach will not work. Thoughts?

Also, we'll need to update to 1.2.1 :)